strongbox-db-import/src/main/java/org/carlspring/strongbox/dbimport/Application.java

```diff
@@ -1,19 +1,23 @@
-package org.carlspring.strongbox.db;
+package org.carlspring.strongbox.dbimport;
```

Comment:
Cosmetic: Perhaps a better package name would be `org.carlspring.strongbox.db.import`?

Reply:
this `dbimport` package name prevents Spring to autoscan `EmbeddedDbServerConfiguration` from `strongbox-data-service`.

Reply:
Thanks for clarifying! :)